.github/workflows/chart-release-manual.yml

```diff
@@ -0,0 +1,27 @@
+name: Push Charts to helm-charts-registry manually with specified version
+
+on:
+  workflow_dispatch:
+    inputs:
+      chart_version:
+        type: string
+        description: "Chart version"
+        required: true
+
+concurrency:
+  group: dbildungs-iam-server-chart-manual
+  cancel-in-progress: true
+
+jobs:
+  scan:
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-helm-kics.yaml@1.3.0
+    permissions:
+      contents: read
+  release:
+    needs: scan
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/chart-release.yaml@1.3.0
+    secrets: inherit
+    with:
+      chart_name: dbildungs-iam
```

Comment:
think it should be dbildungscloud-iam-server

Reply:
Planned for later, probably together with moving the keycloak subchart into it's own repository in the next PI topic.